src/cmmvae/runners/cli.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, extra_parser_kwargs: dict = {}, **kwargs):
         Handles loading trainer, model, and data modules from config file,
         while linking common arguments for ease of access.
         """
-        self.is_run = not kwargs.get("run", False)
+        self.is_run = not kwargs.get("run", True)
```

Comment:
Logic doesn't follow LightningCli precedent where run signify's running sub command. When is_run = true no ckpt_path should be added as the fit subcommand already expects this. This change also makes it so by default the ckpt_path will be added which shouldn't be the case when running a built in subcommand. Am I missing something but this feature already exists by specifying run=False?

Reply:
`self.is_run` isn't passed to the LightningCli `init()` here.
In the old implementation of the cli on main, the adding of `--ckpt_path` to the parser happens as `if not self.is_run`.
In the previous version where `self.is_run = not kwargs.get("run", False)`, `--ckpt_path` would always be skipped as that condition is always `False` in the default case. Setting `run=False` results in no change from the default behavior with this implementation either.
The condition on whether to add `--ckpt_path` to the parser was changed to `if self.is_run`, which will be `True` when run is provided and set to `False`. In the default case where a sub-command is being run, this should be set to `True`, or not provided and default to `True`.
Looking at your dev branch, it seems this issue was already addressed alongside your other changes in a more elegant manner.
Your comments highlight a larger issue where the correlations rule should just be implemented as a subcommand, or not use the cli at all. There is also an issue where the model does not seem to be actually loading form the provided checkpoint right now.